### PR TITLE
Flo's December updates

### PR DESCRIPTION
**Pull request by NolwennP**
- [x] Link to doc

### CODE REVIEW

File: content/updates/2020-12/1-Easily-measure-your-product-quality.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+# Easily measure your product quality
+::: meta-data type="New" features="Quality" available="early December" link-to-doc="../articles/understand-data-quality.html#about-data-quality-insights"
+
+Measuring your product quality has never been so easy thanks to our new simplified way to score your product: the **quality score**. This score replaces the enrichment and consistency grades for the sake of clarity.
+
+![The new quality score](../img/quality-score-in-pef.png) 
+
+You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to filter on it to focus on the products that need your attention due to their low quality scoring.
```
Comment (LaureBro) [inline content/updates/2020-12/1-Easily-measure-your-product-quality.md:16]:
```suggestion
You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to filter on it to focus on the products that need your attention due to their low-quality scoring.
```

File: content/updates/2020-12/2-Revamped-data-quality-dashboard.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# A revamped data quality dashboard
+::: meta-data type="Improvement" features="Quality" available="early December" link-to-doc="../articles/improve-data-quality.html#know-your-data-quality"
+
+Under the `Activity` menu, discover a totally reworked `Data Quality Insights` dashboard to easily follow the catalog's data quality. Besides the UX improvement, a new chart and key indicators are here to help you get a better overview of where you are and the progress you made so far on this crucial topic: the quality of your product information. 
+
+![The revamped DQI dashboard](../img/new-dqi-dashboard.png) 
+
+A step chart now shows the overall quality score evolution on a monthly basis.  
+New key indicators are also displayed to draw your attention to specific quality criteria, such as the percentage of products without images. Those indicators being taken into account in the quality score, don't hesitate to use them to drive your product enrichment and reach a better quality score.
```
Comment (LaureBro) [inline content/updates/2020-12/2-Revamped-data-quality-dashboard.md:17]: Should we specify that "products without images" relate to asset attributes?
```suggestion
New key indicators are also displayed to draw your attention to specific quality criteria, such as the percentage of products without images. Those indicators are taken into account in the quality score, don't hesitate to use them to drive your product enrichment and reach a better quality score.
```

File: content/updates/2020-12/3-Study-your-product-quality-score-in-your-BI-tools.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Deliver your product quality scores to other applications
+::: meta-data type="New" features="Quality, Connectivity" available="early December" link-to-doc="TODO"
+
+Are interested in sending your product quality score to third-party applications to compare this score to relevant metrics, such as the number of sales or the number of returns? Or maybe, you want to know, on your website, which products should be highlighted according to the quality of their enrichment level. Well, great news! It is now possible thanks to an evolution of our Rest API.
```
Comment (LaureBro) [inline content/updates/2020-12/3-Study-your-product-quality-score-in-your-BI-tools.md:10]:
```suggestion
Are you interested in sending your product quality score to third-party applications to compare this score to relevant metrics, such as the number of sales or the number of product returns? Or maybe, you'd like to know on your website, which products should be highlighted according to the quality of their enrichment level? Well, great news! It is now possible thanks to an evolution of our Rest API. 
```

File: content/updates/2020-12/90-Discover-new-designs.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+pim_announcement_alt_img: "The revamped locales grid"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Discover new designs
+::: meta-data type="Improvement" features="Productivity" available="late December"
+
+To always increase your productivity and the level of lisibility of our PIM, we redesign some parts of the interface: the login page, the flash messages and the management page for our dear locales. Enjoy!
```
Comment (LaureBro) [inline content/updates/2020-12/90-Discover-new-designs.md:12]:
```suggestion
To always increase your productivity and the level of readability of our PIM, we redesigned some parts of the PIM interface: the login page, flash messages, and the management page of our dear locales. Enjoy!
```

**Review by fleguellec — APPROVED** [review]
Perfect! I couldn't have written it better.

File: content/updates/2020-12/2-Revamped-data-quality-dashboard.md (hunk 2 above)
Comment (NolwennP) [reply content/updates/2020-12/2-Revamped-data-quality-dashboard.md]: Not only asset attributes, but also basic image attribute. ;)

**Review by LaureBro — APPROVED** [review]
approved (but one missing article link for # Deliver your product quality scores to other applications). thanks!

File: content/updates/2020-12/3-Study-your-product-quality-score-in-your-BI-tools.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Deliver your product quality scores to other applications
+::: meta-data type="New" features="Quality, Connectivity" available="mid- December" link-to-doc="TODO"
```
Comment (LaureBro) [inline content/updates/2020-12/3-Study-your-product-quality-score-in-your-BI-tools.md:8]:
```suggestion
::: meta-data type="New" features="Quality, Connectivity" available="mid-December" link-to-doc="TODO"
```

**Review by lucasj44000 — APPROVED** [review]

**Pull request merged**